src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -395,4 +404,72 @@ public function theyShouldHaveAnAccountCreated(CustomerInterface $customer)
             'Customer should have an account, but they do not.'
         );
     }
+
+    /**
+     * @When I view details of the customer :customer
+     */
+    public function iViewDetailsOfTheCustomer(CustomerInterface $customer)
+    {
+        $this->showPage->open(['id' => $customer->getId()]);
+    }
+
+    /**
+     * @Then his name should be :name
+     */
+    public function hisNameShouldBe($name)
+    {
+        Assert::same(
+            $name,
+            $this->showPage->getCustomerName(),
+            sprintf('Customer name should be "%s", but it is not.', $name)
```

Comment:
No need for sprintf, `sprintf('Customer name should be "%s", but it is not.', $name)` -> `'Customer name should be "%s", but it is not.'`